drivers/scheduler/k8s/specs/wordpress/pure/pure-storage-class.yaml

```diff
@@ -18,8 +17,7 @@ metadata:
 provisioner: pxd.portworx.com
 parameters:
   # Tests:
-  # * FlashArray Direct Access w/ filesystem
-  # * Default filesystem type (ext4), mount options and create options
-  # * No QoS
-  backend: "pure_block"
+  # * FlashBlade Direct Access
+  # * Default export rules
+  backend: "pure_file"
```

Comment:
Why we want to remove FA DA from the wordpress test?
Why we need two SCs (wordpress-sc-legacy-shared, wordpress-sc) for FBDA?

Reply:
The storage classes are specified as the same for all providers (portworx, azure, aws, etc.) so this file has to provide both.

Right now wordpress has both block and file so we can't split it and run "only FA" or "only FB". To fix that I'm making wordpress FB only and making another mysql app that's FA only. Then, to run FA-only or FB-only tests, we'll use the combo in the PR description.

Reply:
Gocha, thanks for the explanation.